.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "docs/themes/docsy"]
+	path = docs/themes/docsy
+	url = https://github.com/google/docsy
```

Comment:
im a little nervous about adding submodules - it might be FUD on my part but I remember having trouble with them in the past and https://codingkilledthecat.wordpress.com/2012/04/28/why-your-company-shouldnt-use-git-submodules/ confirms that

maybe we can mitigate the issues with some automation tho? it seems like the problems mostly have to do with making changes to the submodule

it does make me wonder if vendoring will just be easier tho